### ISSUE TITLE
Patch 7.4.856 uses plines_nofill which is not always available

### ISSUE THREAD

**Issue by luchr** [vim/vim#415]
Patch 7.4.856 changed `src/move.c` and uses `plines_nofill`. But if you look at `src/misc1.c` you see that this function is only defined 

```
 if defined(FEAT_DIFF) || defined(PROTO)
```

Hence if you try to compile a minimal vim you get the error message

```
 objects/move.o: In function `scroll_cursor_top':
 /vim74/src/move.c:1772: undefined reference to `plines_nofill'
 collect2: error: ld returned 1 exit status
 link.sh: Linking failed
```

For example I get this message if I use

```
 configure --enable-luainterp=no  --disable-mzschemeinterp --enable-perlinterp=no \
 --enable-pythoninterp=no  --enable-python3interp=no  --disable-tclinterp  \
 --enable-rubyinterp=no  --disable-cscope   --disable-workshop  --disable-netbeans \
 --disable-sniff  --enable-multibyte  --disable-hangulinput   --disable-xim   \
 --disable-fontset  --enable-gui=no   --with-features=tiny 
```

BTW. There are other undef references, too:

```
 /vim74/src/ex_docmd.c:4742: undefined reference to `qf_get_size'
```

That's because of patch 7.4.858.

**Comment by BlackIkeEagle** [comment]
there is a fix for that in #416 and additional fix in the mailinglist to fix 858 too

**Comment by chrisbra** [comment]
fixed thanks for contributing
